src/components/ebay-combobox/component.js

```diff
@@ -121,13 +127,12 @@ export default {
         }
 
         if (
-            this.expander &&
-            this.expander.expanded &&
+            this.isExpanded() &&
             !wasClickedOption &&
             !this.buttonClicked &&
             this.input.expanded !== true
         ) {
-            this.expander.expanded = false;
+            this.collapse();
         }
```

Comment:
A little tangential to this PR: I think that the parts that touch `input.expanded` should have adequate comments why they are necessary.